### PR TITLE
Feature/25630 colour contrast applied

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: pupil-spa/src/app/familiarisation-colour/familiarisation-colour.component.ts [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+    this.selectedContrast = selectedContrast;
+  }
 
   onClick() {
-    this.router.navigate(['sign-in-success']);
+    const accessArrangements = new AccessArrangements;
+    accessArrangements.contrast = this.selectedContrast || 'bow';
+    this.storageService.setItem(accessArrangementsDataKey, accessArrangements);
+
+    this.router.navigate(['access-settings']);
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/familiarisation-colour/familiarisation-colour.component.ts]: @danwilliams like I mentioned on #887 I would expect the navigation to differ in the following two scenarios:
- User is choosing colour contrast pref after login (should navigate to welcome page)
- User is editing colour contrast pref from access settings page (should navigate back to access settings page)

Comment (danwilliams) [reply pupil-spa/src/app/familiarisation-colour/familiarisation-colour.component.ts]: That's resolved in #887

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/familiarisation-colour/familiarisation-colour.component.ts]: @danwilliams 👍

**Comment by GuyHarwood** [comment]
Closing as open for 12 days.  Please resolve conflicts and re-raise when you have none open

**Comment by danwilliams** [comment]
@GuyHarwood this is waiting for PR #887 to get past Travis - apparently it's a Travis problem (Mo has been in contact with them). Until then, this PR cannot move along, as it needs 25629 to be merged into master and then this 25630 to be rebased. Closing until then seems fine to me 👍

**Comment by danwilliams** [comment]
Rebased using latest master - dependencies now resolved.

**Comment by jon-shipley** [comment]
@danwilliams - please can you provide a PR description?

**Pull request merged**